config/application.rb

```diff
@@ -31,5 +31,6 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
+    config.web_console.whitelisted_ips = '192.87.85.209'
```

Comment:
oh ok yeah it should be fine we can always remove it if it causes problems in production